spec/commands/install_spec.rb

```diff
@@ -321,6 +321,28 @@
       expect(File.exist?(bundled_app("Gemfile.lock"))).to eq(true)
     end
 
+    it "throws an error if a gem is added twice in Gemfile without version requirements" do
+      install_gemfile <<-G
+        source "file://#{gem_repo2}"
+        gem "rack"
+        gem "rack"
+      G
+
+      expect(out).to include("Your Gemfile lists the gem rack (>= 0) more than once.")
+      expect(out).to include("You should probably keep only one of them")
```

Comment:
Clarify the message and be more direct:
`Remove any duplicate entries and specify the gem only once (per group).`